src/diffusers/loaders/textual_inversion.py

```diff
@@ -561,6 +561,7 @@ def unload_textual_inversion(
                 tokenizer._added_tokens_encoder[token.content] = last_special_token_id + key_id
                 key_id += 1
         tokenizer._update_trie()
+        tokenizer._update_total_vocab_size()
```

Comment:
Would be nice to add a comment explaining this change.

Reply:
added simple comment for that